x-pack/performance/journeys/apm_service_inventory.ts

```diff
@@ -35,7 +35,7 @@ export const journey = new Journey({
     );
   },
   ftrConfigPath: 'x-pack/performance/configs/apm_config.ts',
-  skipped: true
+  skipped: true,
```

Comment:
I didn't edit this file, I think it was pulled in with an auto-commit after my latest merge upstream